incubator/traefik/README.md

```diff
@@ -0,0 +1,82 @@
+# Traefik
+
+[Traefik](http://traefik.io/) is a modern HTTP reverse proxy and load balancer made to deploy
+microservices with ease.
+
+## Introduction
+
+This chart bootstraps Traefik as a Kubernetes ingress controller with optional support for SSL and
+Let's Encrypt. By default, Traefik will be installed into the `kube-system` namespace.
```

Comment:
Is this a requirement? I think we generally prefer that Helm manages the namespace Charts get installed into with the `--namespace` flag.